main.py

```diff
@@ -1,7 +1,46 @@
+from django.db.models import QuerySet
+
 import init_django_orm  # noqa: F401
 
-from django.db.models import QuerySet
+from db.models import Actor, Genre
 
 
 def main() -> QuerySet:
-    pass
+    list_of_genre_name = ["Western", "Action", "Dramma"]
+    dict_actors = [
+        {"first_name": "George", "last_name": "Clooney"},
+        {"first_name": "Keanu", "last_name": "Reeves"},
+        {"first_name": "Scarlett", "last_name": "Keegan"},
+        {"first_name": "Will", "last_name": "Smith"},
+        {"first_name": "Jaden", "last_name": "Smith"},
+        {"first_name": "Scarlett", "last_name": "Johansson"}
+    ]
```

Comment:
According to the checklist, it's recommended to use a list of tuples for better iterating. Consider changing `dict_actors` to a list of tuples to improve code readability and maintainability.